### PR TITLE
chore: update Pipfile.lock

### PR DESCRIPTION
**Pull request by vjousse**
## :wrench: Problem

Due to the merge of https://github.com/MTES-MCT/ecobalyse/pull/608 the deploy on scalingo is failing https://github.com/MTES-MCT/ecobalyse/pull/635

It seems that the `Pipfile.lock` was not up to date (but for a reason I don't get the build on the merged PR was not failing).

## :cake: Solution

Update the `Pipfile.lock` using `pipenv install` locally.

## :desert_island: How to test

The scalingo build of this PR should be green.

### CODE REVIEW

**Review by n1k0 — APPROVED** [review]
I reviewed each sha one by one and they're all great. Kidding, I'm just gonna trust you on this one 😅

File: Pipfile.lock [hunk 1]
```diff
@@ -66,55 +153,80 @@
...
+                "sha256:cf5d1c9e6837f8af9f92b6bd3e86d513cdc11f60fd62185cc49ec7d1aba34864",
+                "sha256:e61155fae27570692ad1d327e81c6cf27d535a5d7ef97648a17d922224b216de",
+                "sha256:e7f387600d424f91576af20518334df3d97bc76a300a755f9a8d6e4f5cadd289",
+                "sha256:ed08d2703b5972ec736451b818c2eb9da80d66c3e84aed1deeb0c345fefe461b",
+                "sha256:fbd6acc766814ea6443628f4e6751d0da6593dae29c08c0b2606164db026970c",
+                "sha256:feff59f27338135776f6d4e2ec7aeeac5d5f7a08a83e80869121ef8164b74af9"
             ],
             "markers": "python_version >= '3.10'",
-            "version": "==1.26.4"
+            "version": "==2.0.0"
```
Comment (ccomb) [inline Pipfile.lock:221]: This has not yet been tested with numpy 2.0, maybe we shoul stick with the latests 1.x version for now

**Review by ccomb — CHANGES_REQUESTED** [review]

File: Pipfile.lock (hunk 1 above)
Comment (vjousse) [reply Pipfile.lock]: Good catch, I suppose that it's the introduction of `pandas` that is pulling the latest `numpy` version. I've sticked the `numpy` version to `1.26.4`.

**Review by ccomb — APPROVED** [review]
:+1:

**Pull request merged**